### ISSUE TITLE
[BUG] Plots.jl v1.22.1 generates warnings under vscode sessions

### ISSUE THREAD

**Issue by BoundaryValueProblems** [JuliaPlots/Plots.jl#3831]
When I run `using Plots` under my vscode, I got the following warning.
This does not happen under my simple terminal session.
Also, v1.22.0 did not have any problems like this.
It would be great if this problem is fixed.
Thanks!
## Details
┌ Warning: Error requiring `UnicodePlots` from `Plots`
│   exception =
│    LoadError: UndefVarError: UserColorType not defined
│    Stacktrace:
│      [1] getproperty(x::Module, f::Symbol)
│        @ Base ./Base.jl:26
│      [2] top-level scope
│        @ ~/.julia/packages/Plots/1WwM3/src/backends/unicodeplots.jl:72
│      [3] include(mod::Module, _path::String)
│        @ Base ./Base.jl:386
│      [4] include(x::String)
│        @ Plots ~/.julia/packages/Plots/1WwM3/src/Plots.jl:1
│      [5] top-level scope
│        @ ~/.julia/packages/Plots/1WwM3/src/init.jl:81
│      [6] eval
│        @ ./boot.jl:360 [inlined]
│      [7] eval
│        @ ~/.julia/packages/Plots/1WwM3/src/Plots.jl:1 [inlined]
│      [8] (::Plots.var"#309#345")()
│        @ Plots ~/.julia/packages/Requires/7Ncym/src/require.jl:99
│      [9] err(f::Any, listener::Module, modname::String)
│        @ Requires ~/.julia/packages/Requires/7Ncym/src/require.jl:47
│     [10] (::Plots.var"#308#344")()
│        @ Plots ~/.julia/packages/Requires/7Ncym/src/require.jl:98
│     [11] withpath(f::Any, path::String)
│        @ Requires ~/.julia/packages/Requires/7Ncym/src/require.jl:37
│     [12] (::Plots.var"#307#343")()
│        @ Plots ~/.julia/packages/Requires/7Ncym/src/require.jl:97
│     [13] #invokelatest#2
│        @ ./essentials.jl:708 [inlined]
│     [14] invokelatest
│        @ ./essentials.jl:706 [inlined]
│     [15] foreach(f::typeof(Base.invokelatest), itr::Vector{Function})
│        @ Base ./abstractarray.jl:2141
│     [16] loadpkg(pkg::Base.PkgId)
│        @ Requires ~/.julia/packages/Requires/7Ncym/src/require.jl:27
│     [17] #invokelatest#2
│        @ ./essentials.jl:708 [inlined]
│     [18] invokelatest
│        @ ./essentials.jl:706 [inlined]
│     [19] _tryrequire_from_serialized(modkey::Base.PkgId, build_id::UInt64, modpath::String)
│        @ Base ./loading.jl:715
│     [20] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String)
│        @ Base ./loading.jl:771
│     [21] _tryrequire_from_serialized(modkey::Base.PkgId, build_id::UInt64, modpath::String)
│        @ Base ./loading.jl:711
│     [22] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String)
│        @ Base ./loading.jl:771
│     [23] _require(pkg::Base.PkgId)
│        @ Base ./loading.jl:1020
│     [24] require(uuidkey::Base.PkgId)
│        @ Base ./loading.jl:936
│     [25] require(into::Module, mod::Symbol)
│        @ Base ./loading.jl:923
│     [26] eval
│        @ ./boot.jl:360 [inlined]
│     [27] eval
│        @ ./Base.jl:39 [inlined]
│     [28] repleval(m::Module, code::Expr, #unused#::String)
│        @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.4.3/scripts/packages/VSCodeServer/src/repl.jl:157
│     [29] (::VSCodeServer.var"#69#71"{Module, Expr, REPL.LineEditREPL, REPL.LineEdit.Prompt})()
│        @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.4.3/scripts/packages/VSCodeServer/src/repl.jl:123
│     [30] with_logstate(f::Function, logstate::Any)
│        @ Base.CoreLogging ./logging.jl:491
│     [31] with_logger
│        @ ./logging.jl:603 [inlined]
│     [32] (::VSCodeServer.var"#68#70"{Module, Expr, REPL.LineEditREPL, REPL.LineEdit.Prompt})()
│        @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.4.3/scripts/packages/VSCodeServer/src/repl.jl:124
│     [33] #invokelatest#2
│        @ ./essentials.jl:708 [inlined]
│     [34] invokelatest(::Any)
│        @ Base ./essentials.jl:706
│     [35] macro expansion
│        @ ~/.vscode/extensions/julialang.language-julia-1.4.3/scripts/packages/VSCodeServer/src/eval.jl:34 [inlined]
│     [36] (::VSCodeServer.var"#53#54")()
│        @ VSCodeServer ./task.jl:411
│    in expression starting at /Users/saito/.julia/packages/Plots/1WwM3/src/backends/unicodeplots.jl:72
└ @ Requires ~/.julia/packages/Requires/7Ncym/src/require.jl:49

### Backends

This bug occurs on ( insert `x` below )

Backend      | yes | no  | untested
-------------|-----|-----|---------
gr (default) |  x   |     |
pyplot       |   x  |     |
plotly       |   x  |     |
plotlyjs     |  x   |     |
pgfplotsx    | x   |     |
inspectdr    |  x  |     |

### Versions

Plots.jl version: v1.22.1
Backend  version (`]st -m <backend(s)>`):
(@v1.6) pkg> st -m GR
      Status `~/.julia/environments/v1.6/Manifest.toml`
  [28b8d3ca] GR v0.59.0

(@v1.6) pkg> st -m PyPlot
      Status `~/.julia/environments/v1.6/Manifest.toml`
  [d330b81b] PyPlot v2.10.0

(@v1.6) pkg> st -m PlotlyJS
      Status `~/.julia/environments/v1.6/Manifest.toml`
  [f0f68f2c] PlotlyJS v0.18.7

Output of `versioninfo()`:
Julia Version 1.6.2
Commit 1b93d53fc4 (2021-07-14 15:36 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin18.7.0)
  CPU: Intel(R) Core(TM) i7-5557U CPU @ 3.10GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.1 (ORCJIT, broadwell)
Environment:
  JULIA_EDITOR = code
  JULIA_NUM_THREADS = 

This problem also happens on my Windows machine.

**Comment by t-bltg** [comment]
What is your `UnicodePlots` version:
`(@v1.6) pkg> st UnicodePlots`
?

**Comment by BoundaryValueProblems** [comment]
Thanks, @t-bltg. Here is the output:
```
(@v1.6) pkg> st -m UnicodePlots
      Status `~/.julia/environments/v1.6/Manifest.toml`
  [b8865327] UnicodePlots v1.4.0
```

**Comment by t-bltg** [comment]
What's the output of:
`(@v1.6) pkg> add UnicodePlots@2.4.4`
?

**Comment by BoundaryValueProblems** [comment]
OK, due to my use of the https://github.com/joshday/AverageShiftedHistograms.jl, if I try to add the more recent version of `UnicodePlots`, it gets the following error. I think I should ask the developer of AverageShiftedHistorgrams.jl to update its `Project.toml`.
```
(@v1.6) pkg> add UnicodePlots#master
     Cloning git-repo `https://github.com/JuliaPlots/UnicodePlots.jl.git`
    Updating git-repo `https://github.com/JuliaPlots/UnicodePlots.jl.git`
   Resolving package versions...
ERROR: Unsatisfiable requirements detected for package UnicodePlots [b8865327]:
 UnicodePlots [b8865327] log:
 ├─possible versions are: 2.4.4 or uninstalled
 ├─UnicodePlots [b8865327] is fixed to version 2.4.4
 └─found to have no compatible versions left with AverageShiftedHistograms [77b51b56]
   └─AverageShiftedHistograms [77b51b56] log:
     ├─possible versions are: 0.7.0-0.8.6 or uninstalled
     └─restricted to versions 0.8.6-0.8 by WaveletsExt [8f464e1e], leaving only versions 0.8.6
       └─WaveletsExt [8f464e1e] log:
         ├─possible versions are: 0.1.8 or uninstalled
         └─WaveletsExt [8f464e1e] is fixed to version 0.1.8
```
